src/components/Dashboard.js

```diff
@@ -120,18 +121,22 @@ const AllScores = () => {
 	if (loadinG) return <div>Loading...</div>
 	if (errorR) return `Error! ${errorR.message}`
 	if (erroR) return `Error! ${errorR.message}`
-	label = dataR['Users'].map(score => {
-		return score.user_tests_rel[0].Test.score
+	label = dataR['UserTests'].map(score => {
+		return score.Test.score
+	})
+	dates = dataR['UserTests'].map(date => {
+		return date.created_at
```

Comment:
You need to convert this to local date time, otherwise it will be shown like this 23:26:01.401629+00, which isn't very user-friendly display.